### PR TITLE
(PDB-2436) create a troubleshooting guide

### PR DESCRIPTION
**Pull request by wkalt**
Add a troubleshooting guide and ERD diagram to docs.

### CODE REVIEW

File: documentation/pdb_support_guide.markdown [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+  and values in the database, add whatever new paths/values are required, and
+  delete any invalidated pairs.
+
+* `deactivate-node`: The deactivate-node command just updates a column in the
+  certnames table, and isn't likely to be the source of performance issues.
+
+### Database
+
+PDB uses PostgreSQL. The only good way to get familiar with the schema is
+to examine an [erd diagram][erd] and investigate for yourself on a running
```
Comment (wkalt) [inline documentation/pdb_support_guide.markdown]: @nfagerlund this is a link to a .png. Is there anything special I need to do to make it look alright on the site?

Comment (nfagerlund) [inline documentation/pdb_support_guide.markdown]: Nope, looks good to me.

**Comment by ajroetker** [comment]
@wkalt Minor markdown comment, I think you need some newlines between the headers `##.. <Header>` and the paragraphs that follow.

File: documentation/pdb_support_guide.markdown [hunk 2]
```diff
@@ -0,0 +1,260 @@
...
+for information on these settings.
+
+### PDB dashboard screenshot
+
+There are a few things to watch for in the PDB dashboard:
+
+* Deep command queue: Under sustainable conditions, the command queue depth
+  should be in the neighborhood of 0-100 most of the time, with occasional
+  spikes allowed. If the command queue is deeper than 10,000 for any extended
+  period of time, your commands are being processed too slow. Causes of slow
```
Comment (mullr) [inline documentation/pdb_support_guide.markdown]: slowly

**Comment by mullr** [comment]
generally lgtm, this will be valuable

File: documentation/pdb_support_guide.markdown [hunk 3]
```diff
@@ -0,0 +1,260 @@
...
+message queue on disk unprocessed, where it waits for the application to take
+it and submit it to the database. Commands may be processed out of order,
+though an approximate ordering by submission time can be loosely assumed.
+
+### Command processing
+PDB processes the queue concurrently with the number of threads specified [in
+configuration][threads]. Commands are processed differently depending on their
+type:
+
+* `store-report`: store-report commands are relatively inexpensive, consisting
```
Comment (kbarber) [inline documentation/pdb_support_guide.markdown]: why hyphenate commands?

Comment (wkalt) [inline documentation/pdb_support_guide.markdown]: yeah, that's probably not correct anymore. I'll fix it.

**Comment by wkalt** [comment]
comments addressed

**Pull request merged**